django_cloud_deploy/cli/io.py

```diff
@@ -80,9 +145,18 @@ def ask(self, prompt=None):
         return input(self._replace_html_tags(prompt, sys.stdout.fileno()))
 
     def getpass(self, prompt=None):
-        "Prompt the user for a password and return the result."
+        """Prompt the user for a password and return the result."""
         return getpass.getpass(prompt)
 
+    @contextlib.contextmanager
+    def progressbar(self, expected_time: int, message: str):
```

Comment:
Could you write a doc string? Also, are you going to check to see if you are writing to a tty in a follow-up CL?

Reply:
Done.